app/assets/stylesheets/helpers/_header.scss

```diff
@@ -510,40 +510,26 @@ $covid-grey: #262828;
   font-weight: 700;
   margin-right: govuk-spacing(2);
   margin-bottom: govuk-spacing(1);
+  color: govuk-colour("white");
```

Comment:
It looks like we're setting the color twice for `global-bar-title`? (once here, and one a few lines below)

Reply:
🤦 thanks, fixed.